vendors/nuvoton/boards/numaker_iot_m487_wifi/CMakeLists.txt

```
@@ -149,6 +150,30 @@ target_sources(
         "${afr_ports_dir}/pkcs11/core_pkcs11_pal.c"
 )
 
+# OTA
+afr_mcu_port(ota)
+target_sources(
+    AFR::ota::mcu_port
+    INTERFACE 
+        "${afr_ports_dir}/ota/aws_ota_pal.c"
+        "${afr_ports_dir}/ota/aws_ota_pal_spim.c"
+)
+target_include_directories(
+    AFR::ota::mcu_port
+    INTERFACE
+            "${AFR_ROOT_DIR}/libraries/abstractions/pkcs11/corePKCS11/source/portable/mbedtls/include"
+            "${AFR_ROOT_DIR}/libraries/abstractions/pkcs11/corePKCS11/source/include"
+            "${AFR_ROOT_DIR}//libraries/3rdparty/pkcs11"
```

Seems like an extra slash has been added which can be removed
```suggestion
            "${AFR_ROOT_DIR}/libraries/3rdparty/pkcs11"
```

---

@aggarw13 , Thanks of your notification. It's fixed in the new commit 4b9bfde.